tests/run/test_snakemake.py

```diff
@@ -0,0 +1,63 @@
+"""Tests for module `zarp.run.snakemake"""
```

Comment:
Tests look good - however, _all_ of them are integration tests. You would also need to provide _unit_ tests, where you test each individual method/function in isolation

Reply:
I added `test_run_subprocess` which I think is a unit test. Please let me know if this is what you meant.